docs/UserGuide.md

```diff
@@ -297,6 +297,14 @@ Example of mail command: `mail colleagues`
 ![mail.PNG](images/user-guide/mail.PNG)
 ![mail2.PNG](images/user-guide/mail2.PNG)
 
+<div markdown="block" class="alert alert-info">
+**:information_source: Note:**<br>
+You might want to set up a default mail app if the command does not do anything.
```

Comment:
The tone seems kinda off to me. I can't put my finger on it. I think something like "If the command does not do anything, it might be that you don't have a default mail app set up."

Reply:
aight ill change it then merge it